doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -143,14 +143,16 @@ management.
 > This chapter should describe considerations about hardware acceleration, like
 device management.
 
-<a name="3.2.1.4"></a>
-#### 3.2.1.4 CPU management
+<a name="3.2.1.2"></a>
+#### 3.2.1.2 CPU Management
 
-> This chapter should describe considerations about CPU management.
+CPU management has policies to determine placement preferences to use for workloads that are sensitive to cache affinity or latency, and so the workloads must not be moved by OS scheduler or throttled by kubelet. Additionally, some workloads are sensitive to differences between physical cores and SMT, while others (like DPDK-based workloads) are designed to run on isolated CPUs (like on Linux with cpuset-based selection of CPUs and isolcpus kernel parameter specifying cores isolated from general SMP balancing and scheduler algorithms).
```

Comment:
```suggestion
CPU management has policies to determine placement preferences to use for workloads that are sensitive to cache affinity or latency, and so the workloads must not be moved by OS scheduler or throttled by kubelet. Additionally, some workloads are sensitive to differences between physical cores and SMT, while others (like DPDK-based workloads) are designed to run on isolated CPUs (like on Linux with cpuset-based selection of CPUs and isolcpus kernel parameter specifying cores isolated from general SMP balancing and scheduler algorithms) or required to run on the same NUMA domain as some of their other resources.
```

Reply:
@CsatariGergely : This is of course correct text, just I'm not sure it fits here well. In this section I was focusing only on CPU mgmt, and other requirements described in their sections. (NUMA) Topology in #2150 already has similar text, and more we could add as needed to clarify. How apps use all of those features together is in #2154 . Please check again and let me know your recommendation.

Reply:
I would keep it for the shake of completeness in this section, but if you really do not like it I can also accept that.
I resolve the conversation and let you to decide ... :)

Reply:
@petorre can you let me know if you want to accept the change, or continue with your text?